Mathlib/Computability/AkraBazzi/AkraBazzi.lean

```diff
@@ -60,16 +58,6 @@ above version with a sum, as it is simpler and more relevant for algorithms.
 open Finset Real Filter Asymptotics
 open scoped Topology
 
-/-!
-### Definition of Akra-Bazzi recurrences
```

Comment:
Deleted because the section no longer does what is promised here. Also, this sectioning docstring already exists in SumTransform.lean